modules/olm-dependencies-caveats.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-understanding-dependency-resolution.adoc
+
+[id="olm-dependency-caveats_{context}"]
+= Dependency caveats
+
+When specifying dependencies, consider the following caveats.
```

Comment:
Same concern as above module. It reads as if it will end with a colon and then followed by a list.

Rev Suggestion
""When specifying dependencies, there are caveats you should consider."